test/tbb/test_arena_constraints.cpp

```diff
@@ -212,3 +212,10 @@ TEST_CASE("Test concurrency getters output for constraints with custom concurren
     c.set_max_threads_per_core(1);
     check_concurrency_level(c);
 }
+
+//! Testing constraints_threads_per_core() reserved entry point
+//! \brief \ref interface \ref error_guessing
```

Comment:
```suggestion
//! \brief \ref error_guessing
```
It is some internal stuff, not interface.